src/api/v1/statuses.ts

```diff
@@ -1297,6 +1450,26 @@ async function removeEmojiReaction(
     activity,
     { preferSharedInbox: true, excludeBaseUris: [new URL(c.req.url)] },
   );
+  if (!owner.account.protected) {
+    const acceptedRelays = await db.query.relays.findMany({
+      where: eq(relays.state, "accepted"),
+      with: {
+        relayServerActor: true,
+      },
+    });
+    await fedCtx.sendActivity(
+      owner,
+      acceptedRelays.map((relay) => ({
+        id: new URL(relay.relayServerActor.iri),
+        inboxId: new URL(relay.relayServerActor.inboxUrl),
+      })),
+      activity,
+      {
+        preferSharedInbox: true,
```

Comment:
We don't have to turn this on as we aren't providing recipients' shared inbox URLs here.